modules/mobile/spec/request/letters_request_spec.rb

```diff
@@ -0,0 +1,325 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+require_relative '../support/iam_session_helper'
+require_relative '../support/matchers/json_schema_matcher'
+
+RSpec.describe 'letters', type: :request do
+  include JsonSchemaMatchers
+  before { iam_sign_in }
+
+  let(:letters_body) do
+    {
+      'data' => {
+        'id' => '69ad43ea-6882-5673-8552-377624da64a5',
+        'type' => 'letters',
+        'attributes' => {
+          'letters' =>
+                  [
+                    {
+                      'name' => 'Commissary Letter',
+                      'letterType' => 'commissary'
+                    },
+                    {
+                      'name' => 'Proof of Service Letter',
+                      'letterType' => 'proof_of_service'
+                    },
+                    {
+                      'name' => 'Proof of Creditable Prescription Drug Coverage Letter',
+                      'letterType' => 'medicare_partd'
+                    },
+                    {
+                      'name' => 'Proof of Minimum Essential Coverage Letter',
+                      'letterType' => 'minimum_essential_coverage'
+                    },
+                    {
+                      'name' => 'Service Verification Letter',
+                      'letterType' => 'service_verification'
+                    },
+                    {
+                      'name' => 'Civil Service Preference Letter',
+                      'letterType' => 'civil_service'
+                    },
+                    {
+                      'name' => 'Benefit Summary Letter',
+                      'letterType' => 'benefit_summary'
+                    },
+                    {
+                      'name' => 'Benefit Verification Letter',
+                      'letterType' => 'benefit_verification'
+                    }
```

Comment:
Does "certificate_of_eligibility" need to be included here? It may not, I just noticed that it was the only letter_type listed in the schema that was not included here.

Reply:
This is because while certificate_of_eligibility is something the service could return, but the VCR cassette used by the test doesn't include it